src/pycram/robot_descriptions/boxy_description.py

```diff
@@ -29,7 +29,8 @@
 
 right_gripper.add_static_joint_states(GripperState.OPEN, {"right_gripper_joint": 0.548})
 right_gripper.add_static_joint_states(GripperState.CLOSE, {"right_gripper_joint": 0.0})
-
+right_gripper.end_effector_type = GripperType.PARALLEL  # current gripper in sim, change later
+right_gripper.opening_distance = 0.548
```

Comment:
I don't think this value is correct. That would mean the distance between the grippers is over half a meter. 
The values given in the static joint chains are meant for the joint itself, which in this case would be radiants for the joint.